.github/workflows/aws-heroku-helper.sh

```diff
@@ -60,6 +60,12 @@ function build_database() {
   print_header "Cloning Hasura Engine";
   git clone https://github.com/hasura/graphql-engine-heroku;
 
+  echo "Patching graphql-engine-heroku/Dockerfile, before:";
+  tail -n 4 graphql-engine-heroku/Dockerfile;
+  sed -ri "s/DATABASE_URL graphql-engine/DATABASE_URL HASURA_GRAPHQL_DATABASE_URL=\$DATABASE_URL graphql-engine/g" graphql-engine-heroku/Dockerfile;
+  echo "Patching graphql-engine-heroku/Dockerfile, after:";
+  tail -n 4 graphql-engine-heroku/Dockerfile;
```

Comment:
Modify the Dockerfile in `https://github.com/hasura/graphql-engine-heroku` to specify which database we'll use by default and to connect to on `graphql-engine` startup.